app/models/labels/plate_double_label_qc_base.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
Why make a `PlateDoubleLabelQcBase` class rather than bunging this code into the `PlateDoubleLabelQc`class? Do you expect to make more subclasses in future?

Reply:
It's done like that for the 96 well plate, wasn't completely sure why, but it is future proof so copied that. What do you think? Should change both 96 and 384 ones to be consistent if we do that. Some risk of 96 plate labels breaking then?

Reply:
I would vote for simplifying them, but don't mind. I think it's fairly safe - we are removing a class so if we search for references of that class should be ok? So search for `PlateLabelXpBase` ?